index.html

```diff
@@ -161,18 +174,24 @@ <h2 class="card-title">
                 </article>
             </div>
         </section>
-        <section class="about-myself" id="about-myself-id">
-            <div class="about-container">
-                <h1 class="about-me">About me</h1>
-                <p>
-                    I can help you build a product , feature or website Look through some of my work and experience! If you
-                    like
-                    what you see and have a project you need coded,
-                    don't hestiate to contact me.
-                </p>        
-                <button type="button" class="resume-button">Get My Resume
-                </button> 
-            </div>           
+        <section class="about-myself" id="about-myself">            
+            <div class="about-me-container">
+                <h1>About me</h1>
+                <img src="assets/img/aboutme-topright.svg" class="topright">
+                <img src="assets/img/aboutme-topright-desktop.svg" class="topright-desktop">
+            </div>
+            <div class="about-me-background">
+                <div class="about-container">
+                    <p>
+                        I can help you build a product , feature or website Look through some of my work and experience! If you
+                        like
+                        what you see and have a project you need coded,
+                        don't hestiate to contact me.
+                    </p> 
+                    <button type="button" class="resume-button">Get My Resume
+                    </button>                
+                </div>    
+            </div>     
```

Comment:
Your about section looks a little bit different from the design for the desktop view. you can consider adding some padding left and right to the block of texts and make up for the difference in spacing😉

your project
<img width="943" alt="your_about" src="https://user-images.githubusercontent.com/64643995/213636632-daf79f77-1be5-48a7-96ed-b157d1f4a642.png">

design
<img width="519" alt="design_about" src="https://user-images.githubusercontent.com/64643995/213636654-18758240-0b02-4f96-a937-1d58eaf60be1.png">